munki_facts.py

```diff
@@ -1,13 +1,13 @@
-#!/usr/bin/python
+#!/usr/local/munki/munki-python
```

Comment:
This needs to be `#!/usr/local/munki/munki` now I believe.

Reply:
Definitely not that!

(and we're back to `/usr/local/munki/munki-python`...)

Reply:
Make up our minds! 🤣

Reply:
As stated, DISREGARD THIS :D Its fine again.